platform/features/imagery/src/controllers/ImageryController.js

```diff
@@ -185,8 +184,7 @@ define(
         ImageryController.prototype.updateHistory = function (datum) {
             if (this.$scope.imageHistory.length === 0 ||
                 !_.isEqual(this.$scope.imageHistory.slice(-1)[0], datum)) {
-
-                var index = _.sortedIndex(this.$scope.imageHistory, datum, 'utc');
+                var index = _.sortedIndex(this.$scope.imageHistory, datum, this.timeKey);
```

Comment:
`this.timeKey` is not correct.  From the time key, you'd have to look up the time value metadata, and then you'd have to look up the source key for time value.  The formatter handles this for you and also handles interpreting the data such that you can get a meaningful numerical value for sorting purposes.